app_dart/lib/src/model/firestore/task.dart

```diff
@@ -94,7 +94,10 @@ class Task extends Document {
   ///
   /// This is a human-readable name, typically a test name (e.g.
   /// "hello_world__memory").
-  String? get taskName => fields!['endTimestamp']!.stringValue!;
```

Comment:
Was this meant to be removed?

Reply:
This line is actually a duplicate of https://github.com/flutter/cocoon/pull/3494/files#diff-bb6959c7655c2a5b97a3f09154743dd42d0c9167cb7643bbe51fbeffca9a96e6L91. 

Here we are updating it to `name` (line 97 below).